src/NzbDrone.Core/ImportLists/Simkl/User/SimklUserSettings.cs

```diff
@@ -19,9 +19,13 @@ public class SimklUserSettings : SimklSettingsBase<SimklUserSettings>
         public SimklUserSettings()
         {
             ListType = (int)SimklUserListType.Watching;
+
+            // ShowType = "";
         }
 
         [FieldDefinition(1, Label = "List Type", Type = FieldType.Select, SelectOptions = typeof(SimklUserListType), HelpText = "Type of list you're seeking to import from")]
         public int ListType { get; set; }
+
+        // public string ShowType { get; set; }
```

Comment:
This is heading in the correct direction, but needs to be an enum and needs the `FieldDefinition` treatment